src/generator-attributes.cs

```diff
@@ -971,6 +971,13 @@ void GenerateUnsupported (StringBuilder builder)
 		builder.AppendLine ("\")]");
 	}
 
+	void GenerateDeprecated (StringBuilder builder)
+	{
+		builder.Append ("[ObsoletedOSPlatform (\"");
+		GeneratePlatformNameAndVersion (builder);
```

Comment:
Looking at the generator diff, the `Message` part of the attribute is gone:

```diff
diff --git a/old/dotnet/ios/generated-sources/ARKit/ARFrame.g.cs b/new/dotnet/ios/generated-sources/ARKit/ARFrame.g.cs
index 21c8f0e..b80f9c6 100644
--- a/old/dotnet/ios/generated-sources/ARKit/ARFrame.g.cs
+++ b/new/dotnet/ios/generated-sources/ARKit/ARFrame.g.cs
@@ -151,16 +151,10 @@ namespace ARKit {
 		#endif
 		}
 		[Export ("hitTest:types:")]
-		#if __IOS__
-		[Advice ("Starting with ios14.0 Use 'ARSession.Raycast' instead.")]
-		#endif
-		[UnsupportedOSPlatform ("ios14.0")]
+		[ObsoletedOSPlatform ("ios14.0")]
 		[UnsupportedOSPlatform ("tvos")]
 		[UnsupportedOSPlatform ("macos")]
-		#if __MACCATALYST__ && !__IOS__
-		[Advice ("Starting with maccatalyst14.0 Use 'ARSession.Raycast' instead.")]
-		#endif
-		[UnsupportedOSPlatform ("maccatalyst14.0")]
+		[ObsoletedOSPlatform ("maccatalyst14.0")]
 		[SupportedOSPlatform ("ios11.0")]
 		[BindingImpl (BindingImplOptions.GeneratedCode | BindingImplOptions.Optimizable)]
 		public virtual ARHitTestResult[] HitTest (CGPoint point, ARHitTestResultType types)
```

The generated attribute should be something like:

```cs
[ObsoletedOSPlatform ("ios14.0", "Use 'ARSession.Raycast' instead.")]
```

Reply:
I believe I've fixed this now.